src/clean_air/data/data_store_access.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accessing the Clean Air Framework Datastore\n",
+    "Data is stored on an object store on the JASMIN cloud as a \"dataset\".\n",
+    "A dataset is composed of all the datafiles that hold the data (e.g. netcdf or CSV files) plus metadata about the dataset.\n",
+    "Metadata can be accessed independently of the data, so that you don't have to download the entire dataset if you don't need it.\n",
+    "\n",
+    "Data objects are provided to represent the stored data.\n",
+    "Datasets are encapsulated by the `DataSet` class, whilst metadata is encapsulated by the `Metadata` class.\n",
+    "Access to stored datasets and metadata are via the `S3FSDataSetStore` and `S3FSMetdataStore` classes respectively.\n",
+    "(`s3fs` is the underlying library used to access the object store)\n",
+    "\n",
+    "Both data access classes provide a similar interface, working with their respective data objects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
```

Comment:
I'm guessing you created this notebook using PyCharm?!

Reply:
How did you guess? 😄